src/models/spaces/populations/mouse-model/mouse-populations-model.ts

```diff
@@ -195,6 +219,19 @@ export const MousePopulationsModel = types
             enabled: self["inheritance.showStudentControlOfInheritance"]
           });
 
+          return buttons;
+        },
+        get graphButtons(): ToolbarButton[] {
+          const buttons = [];
+
+          buttons.push({
+            title: "Scale",
+            value: self.showMaxPoints,
+            action: (val: boolean) => {
+              self.toggleShowMaxPoints();
+            }
+          });
+
```

Comment:
I liked this idea of storing the toolbar buttons in an array model so I recycled it here.

Reply:
:+1: This worked well when I copied the pattern for the other buttons.